rules/windows/network_connection/net_connection_win_susp_external_ip_lookup.yml

```diff
@@ -0,0 +1,94 @@
+title: Suspicious Network Connection to IP Lookup Service APIs
+id: edf3485d-dac4-4d50-90e4-b0e5813f7e60
+related:
+    - id: ec82e2a5-81ea-4211-a1f8-37a0286df2c2
+      type: derived
+status: experimental
+description: Detects external IP address lookups by non-browser process via services such as "api.ipify.org". This could be indicative of potential post compromise internet test activity.
```

Comment:
```suggestion
description: Detects external IP address lookups by non-browser processes via services such as "api.ipify.org". This could be indicative of potential post compromise internet test activity.
```